web-console/monitoring/metrics/metrics.md

```diff
@@ -0,0 +1,35 @@
+# Metrics
+
+![Metrics page with an empty query field](img/metrics-empty.png)
```

Comment:
Can the user expand the query at this point? Would it just show a message that the query needs to be added? Should instead the arrow be disabled until a query is run?

Reply:
good catch, I'll update that